framework/src/timesteppers/FunctionDT.C

```diff
@@ -20,7 +20,7 @@ template<>
 InputParameters validParams<FunctionDT>()
 {
   InputParameters params = validParams<TimeStepper>();
-  params.addParam<std::vector<Real> >("time_t", "The values of t");
+  params.addRequiredParam<std::vector<Real> >("time_t", "The values of t");
   params.addParam<std::vector<Real> >("time_dt", "The values of dt");
```

Comment:
Do we want to create a separate time stepper for prescribing all times for all time steps for the transient? I tend to say no because this time step has the snapping capability already anyway.